tests/test_graphical_units/test_opengl.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from manim import *
+from manim.opengl import *
+from ..utils.testing_utils import get_scenes_to_test
+from ..utils.GraphicalUnitTester import GraphicalUnitTester
+
+with tempconfig({"use_opengl_renderer": True}):
```

Comment:
If I had to guess, I'd think that this context here is actually not necessary. It *should* be enough to set the `use_opengl_renderer` flag only for `test_scene` below.

Reply:
You are correct! Change made in 3f023ae. I thought some of the settings mattered at definition time, but they do not.